packages/lib-utils/package.json

```diff
@@ -36,9 +36,11 @@
     "redux-thunk": "^2.4.1"
   },
   "dependencies": {
+    "@types/uuid": "^8.3.4",
```

Comment:
No need for type related dependencies here, any `uuid` related types we use in our library are automatically included in our library's `d.ts` bundle.